package.json

```diff
@@ -30,6 +31,7 @@
 		"cypress:update-snapshots": "cypress run-ct --env type=base --config screenshotsFolder=cypress/snapshots/base"
 	},
 	"main": "dist/ncvuecomponents.js",
+	"module": "dist/index.module.js",
```

Comment:
Is this an official naming convention?

Reply:
No, it is just to keep webpack config simple